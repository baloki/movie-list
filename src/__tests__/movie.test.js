import React from 'react';
import Movie from '../movie';
import renderer from 'react-test-renderer';
import data from '../__mocks__/omdb-api-response';

describe('Movie', () => {
  it('renders nothing if no title is passed in', () => {
    const tree = renderer
      .create(<Movie />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a movie item', () => {
    const tree = renderer
      .create(<Movie {...data} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
