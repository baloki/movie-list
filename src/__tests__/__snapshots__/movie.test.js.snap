// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie renders a movie item 1`] = `
<div
  className="movie"
>
  <img
    alt=""
    className="movie__poster"
    src="https://m.media-amazon.com/images/M/MV5BNDYxNjQyMjAtNTdiOS00NGYwLWFmNTAtNThmYjU5ZGI2YTI1XkEyXkFqcGdeQXVyMTMxODk2OTU@._V1_SX300.jpg"
  />
  <div
    className="movie__details"
  >
    <h2>
      The Avengers
    </h2>
    <p>
      <strong>
        Date Added to List:
      </strong>
       
      <br />
      <strong>
        IMDB Rating:
      </strong>
       
      8.1
    </p>
  </div>
</div>
`;

exports[`Movie renders nothing if no title is passed in 1`] = `null`;
